src/common/playlists/PlayThumbnail.jsx

```diff
@@ -26,9 +26,11 @@ const Author = ({ user }) => {
   );
 };
 
-const PlayThumbnail = ({ play }) => {
+const PlayThumbnail = ({ play, addRoute }) => {
   const [cover, setCover] = useState(null);
 
+  const getRoute = addRoute ? `plays/${play.id}` : play.id
```

Comment:
Not required. if the route starts with a slash `/` , automatically it will map from the base URL. It means, it will work from both the pages
Sample 
```javascript
<Link to={`/plays/${play.id}`} state={{ id: play.id }}>
```

Reply:
Oops yess. Fixing it